TransferMethod/Sources/AddTransferMethodController.swift

```diff
@@ -102,6 +102,29 @@ final class AddTransferMethodController: UITableViewController {
         presenter.loadTransferMethodConfigurationFields(forceUpdate ?? false)
         hideKeyboardWhenTappedAround()
         self.navigationController?.presentationController?.delegate = self
+        
+        let notificationCenter = NotificationCenter.default
+        notificationCenter.addObserver(self,
+                                       selector: #selector(appMovedToBackground),
+                                       name: UIApplication.willResignActiveNotification,
+                                       object: nil)
+        notificationCenter.addObserver(self,
+                                       selector: #selector(appBecameActive),
+                                       name: UIApplication.didBecomeActiveNotification,
+                                       object: nil)
+    }
+    
+    @objc
+    func appMovedToBackground() {
+        widgets.removeAll()
+        presenter.sectionData.removeAll()
+        tableView.reloadData()
```

Comment:
This would mean that if app moves to background all the data entered by user will be lost, what if user is just trying to get his bank account number and has to move between apps to get that number?

Reply:
Yeah that is a valid scenario. Did not think of this way.